README.md

```diff
@@ -1,79 +1,48 @@
-[![Build Status](https://travis-ci.org/simplabs/ember-test-selectors.svg?branch=master)](https://travis-ci.org/simplabs/ember-test-selectors)
+ember-test-selectors
+==============================================================================
 
-# ember-test-selectors
+[![Latest NPM release][npm-badge]][npm-badge-url]
+[![TravisCI Build Status][travis-badge]][travis-badge-url]
 
-This Ember CLI Addon __removes all HTML 5 data attributes starting with
-`data-test-` in the `production` environment__. That allows using data
-attributes as element selectors in integration and acceptance tests without
-polluting the markup that is delivered to the end user.
+[npm-badge]: https://img.shields.io/npm/v/ember-test-selectors.svg
+[npm-badge-url]: https://www.npmjs.com/package/ember-test-selectors
+[travis-badge]: https://img.shields.io/travis/simplabs/ember-test-selectors/master.svg?label=TravisCI
+[travis-badge-url]: https://travis-ci.org/simplabs/ember-test-selectors
 
-## Installation
+Enabling better element selectors in [Ember.js](http://emberjs.com) tests
 
-```bash
-ember install ember-test-selectors
-```
 
-## Why use `data` attributes as test selectors?
+Features
+------------------------------------------------------------------------------
 
-Integration and acceptance tests usually __interact with and assert on the
-presence of certain elements__ in the markup that an application renders. These
-elements are identified using CSS selectors. Most projects use one of three
-approaches for CSS selectors in tests:
+- Provides a `testSelector()` function to help you select the right elements
 
-### Selectors based on HTML structure
+- Removes attributes starting with `data-test-` from HTML tags and
+  component/helper invocations in your templates for production builds
 
-This approach simply selects elements by their position in the rendered HTML.
-For the following template:
+- Removes properties starting with `data-test-` from your JS objects like
+  component classes for production builds
 
-```html
-<article>
-  <h1>Post Title</h1>
-  <p>Post Body…</p<>
-</article>
-```
+- Automatically binds properties starting with `data-test-` on all components
+  for development/testing builds
```

Comment:
I guess we can leave the _"for development/testing builds"_ as it might confuse people. Basically we're always doing that (regardless of the env) and strip these assignments when `strip` is `true`.